docs/gnuworld/help_mgr/edit_cmd.php

```diff
@@ -69,7 +69,7 @@
 	$zecrc = md5($HTTP_USER_AGENT . $zets . $user_id);
 	echo "<input type=hidden name=ts value=$zets>\n";
 	echo "<input type=hidden name=crc value=$zecrc>\n";
-	echo "<input type=hidden name=langid value=$langid>\n";
+	echo "<input type=hidden name=lang_id value=$lang_id>\n";
```

Comment:
A much smaller fix would be changing $langid to $lang_id on this line; then no changes are necessary for any other files.

Reply:
You need to change on all 3 files.
Some task are related with these 3 pages, and some actions (like add, edit) will not work.

Reply:
You only need to change all files because you rename `name=langid` to `name=lang_id` in the form. If you don't do that, all you would have to do is change:

```echo "<input type=hidden name=langid value=$langid>\n";```

to 

```echo "<input type=hidden name=langid value=$lang_id>\n";```

But I think the bigger issue is here that we use $langid as input instead of $_POST['langid']